cdap-ui/app/cdap/components/FileBrowser/FilePath/index.js

```diff
@@ -65,32 +75,103 @@ export default class FilePath extends Component {
     });
   }
 
-  render() {
+  renderCollapsedDropdown(collapsedLinks) {
     return (
-      <div className="file-path-container">
+      <div className="collapsed-dropdown">
+        <Dropdown
```

Comment:
Is it possible to add to `UncontrolledComponents` and just use that? We could avoid the `isOpen, toggle` stuff from being added to the state.